pkg/util/util_test.go

```diff
@@ -244,3 +244,33 @@ func TestMergeMap(t *testing.T) {
 
 	}
 }
+
+func TestRemoveString(t *testing.T) {
+	slice1 := []string{"val1", "val2", "val3"}
+	slice2 := []string{"val1", "val2", "val3", "val1"}
+	slice3 := []string{"val1", "val2", "val1", "val3"}
+	slice4 := []string{"val2", "val1", "val1", "val3"}
+	tests := map[string]struct {
+		actual      []string
+		removeValue string
+		expected    []string
+	}{
+		"value is at start":                 {slice1, "val1", []string{"val2", "val3"}},
```

Comment:
@akhilerm -- add an case for value that is not present. The function should return the same list, if the item to be removed is not present.

Reply:
@kmova  That test is already present. The test with key `"empty string to be removed"` does the same.